samples/net/wifi/test_certs/install_certs.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python3
+# Copyright (c) 2025, Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import signal
+import subprocess
+import sys
+from importlib.metadata import PackageNotFoundError, version
+
+from packaging.version import parse as parse_version
+
+
+def signal_handler(sig, frame):
+    print('\nScript terminated by user')
+    sys.exit(0)
+
+
+def check_requirements():
+    try:
+        installed_version = version('nrfcloud-utils')
```

Comment:
Why is a generic wifi sample pulling in nrf cloud stuff? This sounds wrong.

Reply:
Though it's named `nrfcloud` it does work with non-nRF boards too, I have tested with STM32 Nucleo board. The reason for using `nrfcloud` util is it already supports TLS credential shell based device access. In the future we can write up a standalone tool part of Zephyr repo itself.

Reply:
As discussed on Discord, removed this commit from this PR, will raise a new PR with a standalone script (part of Zephyr itself).